.github/actions/backport/action.yml

```diff
@@ -0,0 +1,73 @@
+name: Backport
+
+inputs:
+  GITHUB_TOKEN:
+    required: true
+  REPO_LOCATION:
+    required: true
+  TARGET_BRANCH:
```

Comment:
What about checking if target branch exists?

Reply:
Migrated into the re-usable bit

https://github.com/hazelcast/backport/pull/17/commits/23f82cb71bab6677b8dfbc0594dce3f40d031804